rotkehlchen_mock/__main__.py

```diff
@@ -0,0 +1,82 @@
+from gevent import monkey
+
+monkey.patch_all()  # isort:skip
+
+import logging
+from http import HTTPStatus
+from typing import Any
+
+import gevent
+from flask import Response
+
+from rotkehlchen.api.rest import RestAPI, api_response
+from rotkehlchen.api.server import APIServer
+from rotkehlchen.args import app_args
+from rotkehlchen.logging import TRACE, RotkehlchenLogsAdapter, add_logging_level, configure_logging
+from rotkehlchen.rotkehlchen import Rotkehlchen
+from rotkehlchen.server import RotkehlchenServer
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def get_well_formed_response(*_: Any) -> Response:
+    return api_response(
+        {
+            'result': {},
+        },
+        status_code=HTTPStatus.OK,
+    )
+
+
+def patched_rotkehlchen_server_constructor(self: Any) -> None:
+    # mock entry point inherits all the cmd args of normal entry point
+    arg_parser = app_args(
+        prog='rotkehlchen_mock',
+        description='mocked rotki server',
+    )
```

Comment:
when code fits in one line keep it one line

```suggestion
    arg_parser = app_args(prog='rotkehlchen_mock', description='mocked rotki server')
```